crates/provider-sqldb-postgres/wit/deps/postgres/query.wit

```diff
@@ -0,0 +1,102 @@
+package wasmcloud:postgres@0.1.0-draft;
+
+/// Interface for querying a Postgres database
+interface query {
```

Comment:
We should probably just have this interface that uses the connection handle/token as that was described as the main idea here: https://github.com/wasmCloud/wasmCloud/issues/1914#issuecomment-2086514118. Mixing the two interfaces seems a bit odd

Reply:
Sorry could you explain what you mean exactly? Are you saying to remove `managed-query`?

Reply:
Yeah, I think we'll always have a "handle" to a connection managed by a provider/host and that is what is passed in the `query` interface (just like the `bucket` argument in keyvalue). So I don't think we need `managed-query`